tests/unittest/auto_deploy/multigpu/custom_ops/test_sharded_rmsnorm.py

```
@@ -120,18 +122,19 @@ def test_sharded_rmsnorm_functional(mpi_pool_executor):
     """Functional test: verify sharded_rmsnorm produces correct numerical results."""
     torch.manual_seed(0)
     tensor_parallel_size = mpi_pool_executor.num_workers
+    port = get_free_port()
 
     results = mpi_pool_executor.map(
         _run_sharded_rmsnorm_test,
-        *zip(*[(tensor_parallel_size,)] * tensor_parallel_size),
+        *zip(*[(tensor_parallel_size, port)] * tensor_parallel_size),
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)